src/macros.rs

```diff
@@ -41,6 +42,7 @@ macro_rules! object_struct {
     );
 }
 
+#[macro_export]
 macro_rules! object_impl {
```

Comment:
I don't think this macro is worth exporting, instead of exporting it could you change `object_struct!` to just directly say:
``` rust
unsafe impl ::objc::Message for $name { }
```